syntax = "proto3";
package system_map;





/*
CreateSystem返回
*/
message CreateSystemResponse {
    /*
    应用系统 id
    */
    string instanceId = 1 ;
}

/*
CreateSystemApi返回
*/
message CreateSystemResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    system_map.CreateSystemResponse data = 4 ;
}

